lib/templates/java.template

```diff
@@ -0,0 +1,17 @@
+<% _.forEach(packages, function(package) { %>import <%-package %><%-"\n"%><% }); %>
+public class <%= className %> 
+{ 
+<% _.forEach(properties, function(prop) { %><%-"\t"%>public <%-prop.type %> <%- prop.name %>;<%-"\n"%><% });%> 
+//Default constructor
+public <%= className %>(){
+<% _.forEach(properties, function(prop) { %><%-"\t"%>this.<%- prop.name %> = <%- prop.defaultValue %>;<%-"\n"%><% });%> 
+}
+// Constructor with parameters
+public <%= className %>(<% _.forEach(properties, function(prop) { %><%- prop.type %> <%- prop.name %>, <% });%>){
```

Comment:
This adds a unnecesary comma at the end of the last constructor parameter.
Perform a map function returning `${type} ${name}` and an array join with ','